src/Castle.Windsor/Castle.Windsor.csproj

```diff
@@ -1,754 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\buildscripts\common.props"></Import>
```

Comment:
The tests failed because FEATURE_REMOTING was expected to be defined in the tests project and it was not, so the test execution took a different path. Changing the ordering is a workaround to fix it.
For some reason this happened in the test project, but not on main Windsor project.